CHANGELOG.md

```diff
@@ -37,6 +38,33 @@
 - [v0.2.0](#v020)
 - [v0.1.0](#v010)
 
+## [v1.13.0]
+
+> Release date: unreleased/TBD
+
+### Breaking changes
+
+- The `konnect` format for the `convert` command is deprecated and will be
```

Comment:
This was already part of 1.12. Not sure if I'd call it "breaking change" just yet, as prior 1.11 functionality was not broken, but a bunch of warnings are raised in preparation for the actual breaking change that will happen with a release in the future

Reply:
We've raised deprecations in breaking changes previously, I think mostly to make it prominent and to get users to switch over early. Removed it though since it was actually in 1.12.